modules/eventing/pages/eventing-faq.adoc

```diff
@@ -52,15 +52,15 @@ If versioning is important, include it as part of the data architecture.
 * Can I get old and new values of the document inside the Function?
 
 +
-No. We do not support versioning of documents; therefore, this feature is not available out of the box. Though customers can have another ‘Mother’ bucket that stores documents that could be looked up, in order to determine the difference between the current document and the last modified.
+No. We do not support versioning of documents; therefore, this feature is not available out of the box. Though customers can have another bucket, say ‘Mother’ bucket, that stores documents that could be looked up, to determine the difference between the current document and the last modified.
```

Comment:
Address the user as "you" rather than customer. Also, based on the context, a 'Mother' bucket seems to be a parent bucket that contains the version of documents before the current set of changes. You may want to clarify this with an SME.